cmds/shell/cmd.go

```diff
@@ -81,12 +83,42 @@ func Execute(cmd *cobra.Command, args []string) error {
 	var shell engines.Shell
 	tty := isatty.IsTerminal(os.Stdout.Fd())
 
+	command := []string{}
+	if len(args) > 1 {
+		command = args[1:]
+	}
+
 	switch redirectURL.Query().Get("v") {
 	case "1":
-		return errors.New("the shell client doens't yet support v1 shells")
+		if len(command) == 0 {
+			// Default command for v1
+			// as defined in https://github.com/taskcluster/taskcluster-tools/blob/edb67d523a8302313b1046448d7cbfda33c6d196/src/views/Shell/Shell.js#L7-L24
+			command = []string{
+				"sh", "-c",
+				strings.Join([]string{
```

Comment:
You can make multi-line strings with in golang:
```go
const myLongString = `
blablabla
new line
something more
`

```

Reply:
True, I made it as a `strings.Join()` so that it would generate a single line but still be readable in source code :)